examples/gazelle/buf_deps.bzl

```diff
@@ -0,0 +1,11 @@
+load("@rules_buf//buf:defs.bzl", "buf_dependencies")
+
+def buf_deps():
+    buf_dependencies(
+        name = "buf_deps_barapis",
+        deps = ["buf.build/acme/paymentapis:6e230f46113f498392c82d12b1a07b70", "buf.build/acme/petapis:84a33a06f0954823a6f2a089fb1bb82e", "buf.build/envoyproxy/protoc-gen-validate:dc09a417d27241f7b069feae2cd74a0e", "buf.build/googleapis/googleapis:84c3cad756d2435982d9e3b72680fa96"],
```

Comment:
These should probably be generated to separate lines

Reply:
Yes, I would like that too but we don't have any control over this, we just provide the rule, gazelle does the printing and formatting.

Reply:
Now fixed thanks to Robbert